### PR TITLE
#RI-4894 - add rs preview regions

### PR DESCRIPTION
**Pull request by AmirAllayarovSofteq**

### CODE REVIEW

File: redisinsight/api/config/features-config.json [hunk 1]
```diff
@@ -39,6 +39,16 @@
                 "provider": "GCP",
                 "regions": ["asia-northeast1"]
               }
+            ],
+            "redistackPreview": [
```
Comment (ViktarStarastsenka) [inline redisinsight/api/config/features-config.json:43]: redisStackPreview

Comment (ViktarStarastsenka) [reply redisinsight/api/config/features-config.json:43]: what I pointed is that it is not RediStack it is Redis Stack. Let's use the correct wording

Comment (AmirAllayarovSofteq) [reply redisinsight/api/config/features-config.json:43]: @arthosofteq we have badge, which mark region as redis stack preview, Basically the same logic as for triggers and functions badge icon

Comment (arthosofteq) [reply redisinsight/api/config/features-config.json:43]: @ViktarStarastsenka Sorry, question was to Amir. Anyway I suggest to not use spaces in literals. RedisStack - is better
@AmirAllayarovSofteq ok. Now I see. Requirements were changed. Before the general plan was to show different set of regions for users navigated from different components. So approve. :) However I don't understand why we didn't reimplement this part. It confuses me...

**Review by arthosofteq — APPROVED** [review]

File: redisinsight/api/config/features-config.json (hunk 1 above)
Comment (AmirAllayarovSofteq) [reply redisinsight/api/config/features-config.json]: Sorry i did not catch you. What you mean when say "reimplement this part"?

**Review by arthosofteq — APPROVED** [review]
Approve as CO
@romansergeenkosofteq Please take a look at FE part. (not sure about ":global")

File: redisinsight/api/config/features-config.json (hunk 1 above)
Comment (arthosofteq) [reply redisinsight/api/config/features-config.json]: never mind. all good :)

**Review by romansergeenkosofteq — APPROVED** [review]

**Pull request merged**